packages/notification-services-controller/CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.21.0]
```

Comment:
As this controller is currently used in prod can we bump to 1.0.0

Reply:
Actually, this is done on purpose for this one. We haven't reached a true "stable version" yet, see: https://github.com/MetaMask/core/pull/5318#discussion_r1954962354